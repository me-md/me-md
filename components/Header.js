import React from 'react';
import { Dimensions, StyleSheet } from 'react-native';

import { LinearGradient } from 'expo-linear-gradient';

const height = Dimensions.get('window').height;

export function Header(props) {
  return (
    <LinearGradient
      colors={['#004EFF', '#88CCF1']}
      style={styles.header}
    ></LinearGradient>
  );

}

const styles = StyleSheet.create({
  header: {
    height: height * 0.1,
  }
})