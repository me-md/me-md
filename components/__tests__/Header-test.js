import React from 'react';
import renderer from 'react-test-renderer';

import { Header } from '../Header';

it(`should render correctly`, () => {
  const tree = renderer.create(<Header />).toJSON();

  expect(tree).toMatchSnapshot();
});