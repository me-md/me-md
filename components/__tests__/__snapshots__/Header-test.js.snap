// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<ViewManagerAdapter_ExpoLinearGradient
  proxiedProperties={
    Object {
      "colors": Array [
        4278210303,
        4287155441,
      ],
      "endPoint": undefined,
      "locations": undefined,
      "startPoint": undefined,
    }
  }
  style={
    Object {
      "height": 200.1,
    }
  }
/>
`;
